opencv/src/gen/java/org/bytedeco/opencv/opencv_core/Mat.java

```diff
@@ -360,29 +360,49 @@ elements of an array one by one (or elements from multiple arrays that have the
     private Pointer pointer; // a reference to prevent deallocation
```

Comment:
As can be seen from the generated code above, we don't need to add a Javadoc comment for the original function since the one from the header file of OpenCV is already good enough.

Reply:
As you wish, but this OpenCV comment warns about not forgettting to deallocate manually. But that's not the main problem, it's when the data is deallocated too early by the GC.

Reply:
Users of raw pointers will be the ones to know if their buffers are registered with the GC or not. OpenCV can't know about that. It's actually your comment that isn't entirely accurate. It should say that, for example, if the external buffer has a deallocator registered with the GC, the user needs to make sure to keep a reference somewhere to prevent premature deallocation.

Reply:
I think I've improved your warning. Let me know what you think! Thanks